vulture/core.py

```diff
@@ -88,6 +88,15 @@ def read_file(filename):
         raise VultureInputException(err)
 
 
+def _get_suitable_whitelists(defined_imports):
+    path = os.path.dirname(os.path.abspath(__file__))
+    whitelist_dir = os.path.join(path, 'whitelists')
+    all_whitelists = os.listdir(whitelist_dir)
+    defined_imports = [imp + '.py' for imp in defined_imports]
+    whitelist_names = [imp for imp in defined_imports if imp in all_whitelists]
```

Comment:
Turn defined_imports and whitelist_names into sets, compute their intersection, then return the sorted intersection, but with "whitelists/" prepended. This has the advantage that we don't need to know the dir name "whitelists" in two parts of the code.